akka-http/src/main/scala/akka/http/scaladsl/server/directives/FormFieldDirectives.scala

```diff
@@ -20,7 +20,7 @@ trait FormFieldDirectives extends ToNameReceptacleEnhancements {
   import FormFieldDirectives._
 
   /**
-   * Extracts HTTP form fields from the request as a ``Map[String, String]``.
+   * Extracts HTTP form fields from the request g in Scala 2.10, invocations of this method sometimes fail to as a ``Map[String, String]``.
```

Comment:
Whoa, how did this get here... :-\
PR is only about the removals below (will remove this one)